### ISSUE TITLE
Add accessibility section

### ISSUE THREAD

**Issue by LJWatson** [w3c/manifest#860]
In the [WebApps charter](https://www.w3.org/2019/05/webapps-charter.html#scope) it says:
>Where there are implications for implementors, developers, or users, in the areas of accessibility, internationalization, privacy, and security, each specification must have a section that describes relevant benefits, limitations, and best practice solutions for that particular area.>

To this end each spec needs to have an accessibility section. For the avoidance of doubt in the reader, the section should be included even if it indicates there are no known accessibility considerations.

Relates to w3c/webappswg#15

**Comment by aarongustafson** [comment]
Thanks for this. Thinking for a moment about where the manifest & accessibility overlap, I think it makes sense to enumerate the following (at the very least):

* `name` should be exposed as the accessible name of the application, even in the presence of `short_name` (in which case `name` would be the expansion of `short_name`).
* Once `ImageResource` gets the 👍🏻 in TAG review, I plan to add an optional alternative text property to it (name TBD). In the context of `icons` (and possible `shortcuts`) the `name` would suffice as alternative text (making it optional), but in the context of `screenshots` it should be required.
* We should address, in some way, a host OS’s ability to override manifest-described colors to support user-preferred color modes (high contrast, dark, etc.).

It may also be worth considering whether we want to call out that implementors can enable users to rename their applications to be something more relevant to them (but that the default value should be the manifest `name`)

**Comment by aarongustafson** [comment]
Related:

* [ImageResource#7](/w3c/image-resource/issues/7)
* [Manifest#758](/w3c/manifest/issues/758)

**Comment by michael-n-cooper** [comment]
@LJWatson APA would like to know if the pull request addressed the issue.
